config/lint/.checkov.baseline

```diff
@@ -103,6 +136,342 @@
                 }
             ]
         },
+        {
```

Comment:
These are a lot of checks to disable. Please, fix these issues instead of ignoring them.

You can ignore the ones that ask for using digests because this is only an example, and not a production-aimed workload.